backend/database/__init__.py

```diff
@@ -24,35 +25,38 @@ def drop_all_tables(db: sqlalchemy.engine.Connection) -> None:
     ))
 
 def create_all_tables(db: sqlalchemy.engine.Connection) -> None:
+    db.execute(text('''DROP TABLE IF EXISTS visitors'''))
```

Comment:
This line is unnecessary because there is already a drop_all_tables() function that is called before this.

Reply:
Thanks for pointing this out! I have updated the code!